pkg/cmd/attestation/verify/verify_test.go

```diff
@@ -163,12 +169,13 @@ func TestNewVerifyCmd(t *testing.T) {
 			wants: Options{
 				ArtifactPath:     artifactPath,
 				DigestAlgorithm:  "sha256",
+				Hostname:         "github.com",
+				Limit:            30,
 				OIDCIssuer:       verification.GitHubOIDCIssuer,
 				Owner:            "sigstore",
+				PredicateType:    verification.SLSAPredicateType,
```

Comment:
The test case named "Use specified predicate type" in this table test uses the SPDX predicate type.

Reply:
Is there a test too that fails when no predicate type attestations are returned?

Reply:
I have a test that does not set the predicate type specifically to ensure it defaults to "https://slsa.dev/provenance/v1" since this is just testing that the flags are being set correctly. We don't currently have any code validating that all required options are set but I can add some to confirm that the SLSA predicate type, along with owner/repo and digest algorithm are always set as expected.